app/api/views/brief_responses.py

```diff
@@ -122,3 +123,45 @@ def get_brief_response(brief_response_id):
                   .format(brief_response_id, current_user.supplier_code))
 
     return jsonify(brief_response.serialize())
+
+
+@api.route('/brief-response/<int:brief_id>/suppliers', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_suppliers_responded(brief_id):
+    """Get suppliers responded (role=buyer)
+    ---
+    tags:
+      - "Brief Response"
+    security:
+      - basicAuth: []
+    parameters:
+      - name: brief_response_id
+        in: path
+        type: number
+        required: true
+    definitions:
+      BriefResponse:
+        type: object
+        properties:
+          id:
+            type: number
+          data:
+            type: object
+          brief_id:
+            type: number
+          supplier_code:
+            type: number
+    responses:
+      200:
+        description: A brief response on id
```

Comment:
Is this a better description: `Suppliers that have responded to brief_id` ?

Reply:
updated